.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "rules": {
+        "no-console": 2, // Remember, this means error!
+        "camelcase": "warn",
+        "@typescript-eslint/no-explicit-any": "warn",
+        "@typescript-eslint/no-unused-vars": [
```

Comment:
https://typescript-eslint.io/rules/no-unused-vars/

> Extending "plugin:@typescript-eslint/recommended" in an ESLint configuration enables this rule.


Also, `^_` is the default for `argsIgnorePattern`. Am I reading this right?